cg/services/order_validation_service/workflows/microsalt/validation/inter_field/utils.py

```diff
@@ -0,0 +1,35 @@
+from cg.models.orders.sample_base import ContainerEnum
+from cg.services.order_validation_service.errors.sample_errors import OccupiedWellError
+from cg.services.order_validation_service.workflows.microsalt.models.order import MicrosaltOrder
+from cg.services.order_validation_service.workflows.microsalt.models.sample import MicrosaltSample
+
+
+class PlateSamples:
+
+    def __init__(self, order: MicrosaltOrder):
+        self.wells: dict = {}
+        self._initialize_map(order)
+
+    def _initialize_map(self, order: MicrosaltOrder):
```

Comment:
I think a docstring is useful for this map, since the structure is a bit convoluted.